arch/x86/zefi/printf.h

```diff
@@ -0,0 +1,153 @@
+/*
+ * Copyright (c) 2018 Intel Corporation
```

Comment:
2020, no?

Reply:
Actually this was in the tree before, as part of the original x84_64 work (where it was doing the same "logging from pre-Zephyr code", in that case the i386 stub).  But I did modify it, so yeah, I'll update.